internal/api/hooks.go

```diff
@@ -203,7 +200,7 @@ func (a *API) invokeHTTPHook(ctx context.Context, r *http.Request, input, output
 // transaction is opened. If calling invokeHook within a transaction, always
 // pass the current transaction, as pool-exhaustion deadlocks are very easy to
 // trigger.
-func (a *API) invokePostgresHook(ctx context.Context, conn *storage.Connection, input, output any, hookURI string) error {
+func (a *API) invokePostgresHook(ctx context.Context, conn *storage.Connection, input, output any) error {
```

Comment:
Although the parameter is not used here, it seems to me it might still be needed here, but I'm not familiar with the code.